telco-hub/configuration/reference-crs/optional/cert-manager/certManagerClusterIssuer.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: acme-issuer
+spec:
+  # Example based on: https://cert-manager.io/docs/configuration/acme/#creating-a-basic-acme-issuer
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: your-email@example.com
+    privateKeySecretRef:
+      name: acme-account-key
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
```

Comment:
I think the openshift-default class is more general purpose and should work equally well.